### PR TITLE
Update sgid database policy page

### PR DESCRIPTION
**Pull request by gregbunce**
this pr will update the database policy page to better match our current config including: metadata, domains, and the soon-to-be renamed cemetery project.

### CODE REVIEW

**Comment by gregbunce** [comment]
just a fyi: i put this out here as a draft for feedback/suggestions, but i'll probably be gong MIA till thursday with this month's road update chaos.  but, i'll be back.

File: about/policy/sgid/index.md [hunk 1]
```diff
@@ -25,26 +29,23 @@ When adding a new table to the SGID, a corresponding record will be added to the
...
 
+All new database domains will comply with the rule stating, domain values must match domain descriptions. See the [domain document](https://docs.google.com/document/d/12bdtmtv-ZVF9g-lFshbobx8DbKX9LfbL44HawLjFTVw/edit?usp=sharing) for more information on this effort.
+
 The ESRI default Resolution and Tolerance will be used.
 
 - XY Resolution: 0.0001 Meter
 - XY Tolerance: 0.001 Meter
 
+
+
```
Comment (steveoh) [inline about/policy/sgid/index.md:48]: **nitpick**: let's remove the empty line

File: about/policy/sgid/index.md [hunk 2]
```diff
@@ -61,7 +62,9 @@ All data updates being pushed into the SGID will have removed empty geometries a
 
 All edits made to the internal SGID will be captured in `Last update date` in the metadata, in the [SGID Update log](https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=0), and as an update on the AGRC website data page.
 
+If making a schema change, please consider updating any domain descriptions that do not match the domain value. See this [document](https://docs.google.com/document/d/12bdtmtv-ZVF9g-lFshbobx8DbKX9LfbL44HawLjFTVw/edit?usp=sharing) to learn more about this effort.
+
 ## Removing SGID Products
 {: .text-left }
 
-When data or services are deprecated, the [Cemetery](https://github.com/agrc/cemetery) project is used.
+When deprecating data or services in the SGID, please follow the steps outlined in the [Life Cycle](https://github.com/agrc/cemetery) project.
```
Comment (steveoh) [inline about/policy/sgid/index.md:70]: **chore**: we'll need this link

File: about/policy/sgid/index.md [hunk 3]
```diff
@@ -15,7 +15,11 @@ The following applies to data in the internal SGID database.
 ## Adding New Tables
 {: .text-left }
 
-When adding a new table to the SGID, a corresponding record will be added to the [Data Stewardship spreadsheet](https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=1) and a data page will be added to the AGRC website. The data stewardship spreadsheet will contain at a minimum:
+When adding new data to the SGID, the steps outlined in the [Life Cycle](https://github.com/agrc/cemetery) project will be followed.
```
[the remark this answers is not in the document]
Comment (gregbunce) [reply about/policy/sgid/index.md:18]: i just dropped that name in.  i was hoping for something more exciting (see slack #general from this past Saturday).  but, whatever.

File: about/policy/sgid/index.md [hunk 4]
```diff
@@ -15,7 +15,11 @@ The following applies to data in the internal SGID database.
...
+
+Additional, related info:
```
[the remark this answers is not in the document]
Comment (gregbunce) [reply about/policy/sgid/index.md:20]: we could rename the heading on line 15 to make it more clear.

File: about/policy/sgid/index.md (hunk 1 above)
Comment (gregbunce) [reply about/policy/sgid/index.md:48]: good catch

File: about/policy/sgid/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (steveoh) [reply about/policy/sgid/index.md:18]: oh i glanced over that since i assumed it was my poll.

**Pull request merged**